test/unit/request.test.js

```diff
@@ -565,6 +565,54 @@ describe('Request', function () {
         });
     });
 
+    describe('.size', function () {
```

Comment:
Add a comment explaining how the default values for empty requests are arrived at.